src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,124 @@
+                        connection.prepareStatement(GET_REQUEST)) {
+            getStatement.setLong(1, id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getManufacturersInstance(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer with id: "
+                    + id + " from DB", e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allManufacturers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllStatement = connection.createStatement()) {
+            ResultSet resultSet =
+                        getAllStatement.executeQuery(GET_ALL_REQUEST);
+            while (resultSet.next()) {
+                allManufacturers.add(getManufacturersInstance(resultSet));
+            }
+            return allManufacturers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateStatement = connection.prepareStatement(
+                        UPDATE_REQUEST)) {
+            updateStatement.setString(1, manufacturer.getName());
+            updateStatement.setString(2, manufacturer.getCountry());
+            updateStatement.setLong(3, manufacturer.getId());
+            updateStatement.executeUpdate();
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer: "
+                    + manufacturer, e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteStatement = connection.prepareStatement(
+                        DELETE_REQUEST)) {
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() >= 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete manufacturer with id "
+                    + id, e);
+        }
+    }
+
+    private Manufacturer getManufacturersInstance(ResultSet resultSet) {
+        try {
+            Long id = resultSet.getObject("id", Long.class);
+            String name = resultSet.getString("name");
+            String country = resultSet.getString("country");
+            return new Manufacturer(id, name, country);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't process manufacturer data with result set: "
+                    + resultSet, e);
+        }
+    }

```

Comment:
You shouldn't use the try-catch block here, because you catch the same exception higher in your get() and getAll() methods
```suggestion
    private Manufacturer getManufacturersInstance(ResultSet resultSet) throws SQLException {
            Long id = resultSet.getObject("id", Long.class);
            String name = resultSet.getString("name");
            String country = resultSet.getString("country");
            return new Manufacturer(id, name, country);
        }
    }

```